misc/scripts/install-local.sh

```diff
@@ -785,6 +785,15 @@ if ! checks; then
     return 1
 fi
 
+# If priority exists and is required, and also that this package has not been installed before (first time)
+if [[ -n ${priority} && ${priority} == 'required' ]] && ! is_package_installed "${name}"; then
```

Comment:
Wouldn't this be enough?
```suggestion
if [[ ${priority} == 'required' ]] && ! is_package_installed "${name}"; then
```

Reply:
No. We need to check that `priority` exists first, not assume it does.

Reply:
If it doesn't exist, then it is interpreted as blank, which is different from `"required"`

Reply:
I'd like it to still be more explicit.